fiatconnect-api.md

```diff
@@ -2245,6 +2246,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 The `country` field MUST be a [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. The syntax of the `iban` field MUST correspond to that of a valid International
 Bank Account Number.
 
+#### 8.3.2.5. `IfscAccount`
+
+`IfscAccount` is an account schema that represents transfers for `INR` currency.
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	ifsc: `string`,
+	accountNumber: `string`,
+	country: `string`,
+	fiatAccountType: `FiatAccountTypeEnum.IfscAccount`
```

Comment:
@SonalBhanawat This schema represents a bank account; I think we should use `FiatAccountTypeEnum.BankAccount` here instead.